src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  } else if (days >= 3 && days < 6) {
+    return 100;
+  } else if (days === 6) {
+    return 220;
+  } else {
+    return 230;
```

Comment:
Avoid using `else if` and `else` after an `if` that contains a `return` statement. Each `if` can stand alone with its own `return`, simplifying the structure of the code and making it easier to follow.